modules/app.js

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
```

Comment:
- Please remove all disabled linters, remember linters help us ensure that we write code that is clean and easy to maintain